src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/customerSegment-rule.tpl.html

```diff
@@ -0,0 +1,27 @@
+<div class="blade-static __bottom">
+    <button class="btn" ng-click="saveRule()" ng-disabled="!canSave()">
+        <span>Apply</span>
```

Comment:
Please move to localization resource and change to "Confirm" (discussed with Maya and @pushnitsa)

Reply:
done